src/components/forms/InputGroup/InputGroup.types.ts

```diff
@@ -0,0 +1,5 @@
+import { PropsWithChildren } from 'react';
+
+export type InputGroupProps = PropsWithChildren<{
+  divider?: boolean;
```

Comment:
I missed that  🤦 Could you please rename this to `hasDivider`? We are trying to keep bool props prefixed with is/has/should for better readabilty. Also, add it to prop types.

Reply:
✅  the only outstanding thing I didn't implement was support for MultiValue input, since this input can have more than one row.
I think we should add that one in a subsequent version since we'll need to adjust the design accordingly.